tests/integration_tests/processor_resolved_ts_fallback/run.sh

```diff
@@ -18,7 +18,7 @@ function run() {
 	start_tidb_cluster --workdir $WORK_DIR
 	cd $WORK_DIR
 
-	export GO_FAILPOINTS='github.com/pingcap/tiflow/cdc/sink/mysql/SinkFlushDMLPanic=return(true);github.com/pingcap/tiflow/cdc/sink/mq/producer/kafka/SinkFlushDMLPanic=return(true)'
+	export GO_FAILPOINTS='github.com/pingcap/tiflow/cdc/sinkv2/eventsink/txn/mysql/MySQLSinkExecDMLError=return(true);github.com/pingcap/tiflow/cdc/sinkv2/eventsink/mq/dmlproducer/KafkaSinkAsyncSendError=return(true)'
```

Comment:
Can we remove this?

Reply:
No. We expect that the cdc instance can't execute SQLs before gets killed.